Library/Formula/cake.rb

```diff
@@ -0,0 +1,46 @@
+class Cake < Formula
+  desc "Cake (C# Make) is a build automation system with a C# DSL."
+  homepage "http://cakebuild.net/"
+  url "https://www.nuget.org/api/v2/package/Cake/0.5.1"
```

Comment:
Can we use `https://github.com/cake-build/cake/releases/download/v0.5.1/Cake-bin-v0.5.1.zip` or something instead? Looks like it installs the same content.